lib/ui/screens/register_screen.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:summaries_app/ui/styles/app_colors.dart';
+import 'package:summaries_app/ui/widgets/app_app_bar.dart';
+import 'package:mask_text_input_formatter/mask_text_input_formatter.dart';
+import 'package:summaries_app/ui/widgets/app_drawer.dart';
+
+class RegisterScreen extends StatefulWidget {
+  const RegisterScreen({Key? key}) : super(key: key);
+
+  @override
+  _RegisterScreenState createState() => _RegisterScreenState();
+}
+
+class _RegisterScreenState extends State<RegisterScreen> {
+  final _formKey = GlobalKey<FormState>();
+  final TextEditingController _nameController = TextEditingController();
+  final TextEditingController _phoneController = TextEditingController();
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+
+  get onPressed => null;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Column(
+          children: [buildTitle(), buildForm()],
+        ),
+      ),
+    );
+  }
+
+  buildTitle() {
+    return Container(
+      width: 800,
+      height: 250,
+      color: AppColors.darkBlue,
+      padding: EdgeInsets.only(top: 100),
+      child: Column(
+        children: [
+          Text(
+            'Cadastro',
+            style: TextStyle(
+                fontSize: 80,
+                fontWeight: FontWeight.w500,
+                color: AppColors.offWhite),
+          ),
+          SizedBox(
+            height: 10,
+          ),
+          Text(
+            'Cadastre-se para mais conteúdos incríveis',
+            style: TextStyle(
+                fontSize: 15,
+                fontWeight: FontWeight.w500,
+                color: AppColors.offWhite),
+          ),
+        ],
+      ),
+    );
+  }
+
+  buildForm() {
+    var mask = MaskTextInputFormatter(mask: "(##) # ####-####");
+    return Stack(
```

Comment:
substituir o "stack" por um "column" pois não está sendo feita nenhuma sobreposição então o "stack" é meio q inútil aqui